webdav_sync.sh

```diff
@@ -58,6 +58,39 @@ url_decode() {
 }
 
 
+allFiles=''
+currentDir=$remote_dir_name
+
+getFilesRec()
+{
+    currentDirAbs="$remote_host$remote_path_suffix$currentDir"
+
+
+    dirsAndFiles=$(curl --silent --user "$user":"$passwd" $currentDirAbs -X PROPFIND --data '<?xml version="1.0"?>
+    <a:propfind xmlns:a="DAV:">
+    <a:prop>
+    <a:resourcetype />
+    </a:prop>
+    </a:propfind>'  | grep 'href' | sed 's/<\/\?[^>]\+>//g' | sed 's/HTTP\/1.1 200 OK/\n/g' | sed "s/\/owncloud\/remote.php\/dav\/files\/BA99A208-AF10-40AD-9F39-C01D5CEF0EDB\///g" | grep -v '^$')
```

Comment:
AFAIK, `BA99A208-AF10-40AD-9F39-C01D5CEF0EDB` is installation specific hash. So, this will not for anybody else except you.

Reply:
Ah, of course. Fixed it.